addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-config-vars-service.js

```diff
@@ -392,6 +392,10 @@ class EnvironmentConfigVarsService extends Service {
         Key: 'CreatedBy',
         Value: resolvedVars.username,
       },
+      {
+        Key: 'WorkspaceName',
+        Value: resolvedVars.name,
+      },
```

Comment:
Hi Bhanu, have you tested this change?
do we need the comma on row 398?

Reply:
I've tested the change in dev and it's working as expected.

<img width="609" alt="Screen Shot 2022-10-14 at 16 37 29" src="https://user-images.githubusercontent.com/17888860/195939245-b46633af-914f-497d-b6c0-96e68e7f5102.png">

Reply:
Samantha just told me it was ok to leave that comma